code/modules/projectiles/guns/projectile/automatic/c20r.dm

```diff
@@ -78,7 +78,7 @@
 
 /obj/item/part/gun/frame/c20r/sci
 	name = "C20m frame"
-	desc = "A C20m SMG frame. The syndicate's bread and butter, reverse-engineered. Curiously, it has two sets of threads for a barrel in differing sizes."
+	desc = "A reverse engineered C-20r frame. It can be chambered in either .35 Auto or 10mm Magnum."
```

Comment:
```suggestion
	desc = "A reverse engineered C-20r frame. It can be chambered in either 9mm Auto or 10mm Magnum."
```
Noticed this. Lets not unstandardized are standard